src/main/java/pl/javagame/App.java

```diff
@@ -29,19 +29,41 @@ public static void main(String[] args) {
         for (Player lista : playerList) {
             System.out.println("Gracz: " + lista);
         }
-        scanner.close();
-
         dividingGameToRounds(ROUNDS, numberOfPlayers, playerList);
+        scanner.close();
     }
 
     private static void dividingGameToRounds(int rounds, int numberOfPlayers, ArrayList<Player> playerList) {
```

Comment:
czepialstwo ale nazwy komend zawsze w trybie biernym - `divideGameIntoRooms`

Reply:
bardzo proszę o czepialstwo , o to chodzi. dzięki